Lib/idlelib/configdialog.py

```diff
@@ -672,18 +674,18 @@ def create_page_general(self):
             frame
                 frame_run: LabelFrame
                     startup_title: Label
-                    (*)radio_startup_edit: Radiobutton - startup_edit
-                    (*)radio_startup_shell: Radiobutton - startup_edit
+                    (*)startup_editor_on: Radiobutton - startup_edit
+                    (*)startup_shell_on: Radiobutton - startup_edit
```

Comment:
This is trivial, but with the rename (and since this is moving to its own class soon), I had been thinking about the variable being `startup_window` and then the two choices just being `shell` and `editor`.